### ISSUE TITLE
A11y_AzureToolsStorageExplore_Settings_Transfer_ScreenReader: Voice Over not announces Control type of Control(SAS Duration).

### ISSUE THREAD

**Issue by Suraj1200499** [microsoft/AzureStorageExplorer#2301]
GitHubTags: #A11y_AzureToolsStorageExplorer; #A11yMAS; #A11YTCS; #DesktopApp; #MAS1.3.1; #VoiceOver; #Mac; #CELAAssessID_CosmosDB-FY20-Q2; 

**Environment Details:**
Application Name: Azure Tools Storage Explorer
Application Version: 1.11.0
MAC Version: Catalina 10.15 

**Repro Steps:**
1. Launch Storage Explorer
2. Navigate to Setting menu on left pane and click on it.
3. Navigate to Transfers on right pane and  click on it.
4. "Transfers" screen gets opened.
5. Start Voice Over.
6. Navigate to "SAS Duration" control.
7. Observe Voice Over announcement.

**​Actual:​**
Voiceover says, "3, SAS Duration, stepper 3" and does not announce the Control Type of 'SAS Duration' field.
And also Voice Over announces some  extra content for 'SAS Duration' field.

**Expected:**
Voiceover should announce the Control Type of 'SAS Duration' field.
For eg., VO should say, "3, SAS Duration, Spin button editable"

**UserImpact:**
Screen reader users may not be able to know role of the control.

**Comment by JasonYeMSFT** [comment]
"Stepper" is the word that Voice Over decides for this type of control in chromium. I can reproduce this on a generic Chrome browser with any <input type="number"> element. If "spin button editable" is the standard term for this control type, Voice Over should fix this problem.

**Comment by JasonYeMSFT** [comment]
Filed an issue with Chromium. Tracked at https://bugs.chromium.org/p/chromium/issues/detail?id=1032293

**Comment by Suraj1200499** [comment]
@JasonYeMSFT  Since the primary bug#[1032293](https://bugs.chromium.org/p/chromium/issues/detail?id=1032293) is in open State. We will take action on the bug#[2301](https://github.com/microsoft/AzureStorageExplorer/issues/2301) Once the Primary bug is Closed.

**Comment by Alexandria-R** [comment]
Approved for exclusion per discussion.

**Comment by mstechie** [comment]
GitHubTags:#BM-TCS-StorageExplorer-Mac-Sep2019;

**Comment by BhanuPrakashGit** [comment]
GithubTag:#A11ySev2;

**Comment by BhanuPrakashGit** [comment]
GithubTag:#Regressed:06-01-23;#Closed;
Verified the bug in the environment: 
Microsoft Azure Storage Explorer
Version 1.29.2

Bug is fixed and does not repro. Hence closing the bug.
PFA for reference
[2301_Closed - Trim (2).zip](https://github.com/microsoft/AzureStorageExplorer/files/11624732/2301_Closed.-.Trim.2.zip)
